underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -80,15 +80,24 @@ public SwiftUnderFileSystem(AlluxioURI uri, Configuration configuration) {
     String containerName = uri.getHost();
     LOG.debug("Constructor init: {}", containerName);
     AccountConfig config = new AccountConfig();
-    config.setUsername(configuration.get(Constants.SWIFT_USER_KEY));
-    config.setTenantName(configuration.get(Constants.SWIFT_TENANT_KEY));
-    config.setPassword(configuration.get(Constants.SWIFT_API_KEY));
+    if (configuration.containsKey(Constants.SWIFT_API_KEY)) {
```

Comment:
@yupeng9 apikey is the same value as password. The only change here is to rename configuration key. To call it apikey was a mistake, we should call it as password. But it's the same value...just changing the way we name this value.